.github/workflows/deployment.yml

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) 2023 Files Community
+# Licensed under the MIT License. See the LICENSE.
+
+# Abstract:
+#   This workflow is triggered manually when the maintainer wants
+#   to deploy Preview or Stable to the Azure Blob Storage.
+
+name: Files Deployment
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Select a branch'
+        required: false
+        default: 'Preview'
+        type: choice
+        options:
+          - Preview
+          - Stable
+
+env:
+  SOLUTION_NAME: 'Files.sln'
+  PACKAGE_PROJECT_DIR: 'src\Files.App (Package)'
+  PACKAGE_PROJECT_PATH: 'src\Files.App (Package)\Files.Package.wapproj'
+  TEST_PROJECT_PATH: 'tests\Files.InteractionTests\Files.InteractionTests.csproj'
+  CONFIGURATION: 'Preview' # NOTE: This is just placeholder
+  PLATFORM: 'x64'
+  APPX_BUNDLE_PLATFORMS: 'x64|arm64'
+  WORKING_DIR: ${{ github.workspace }} # Default: D:\a\Files\Files\
+  ARTIFACTS_STAGING_DIR:  ${{ github.workspace }}\artifacts
+  APPX_PACKAGE_DIR: ${{ github.workspace }}\artifacts\AppxPackages
+  PACKAGE_DEPLOYMENT_BRANCH: 'Preview' # NOTE: This is just placeholder
+  PACKAGE_DEPLOYMENT_BRANCH_LOWERCASE: 'preview' # NOTE: This is just placeholder
```

Comment:
The release version needs to use `stable`.

Reply:
This is just placeholder as written.
https://github.com/files-community/Files/pull/14218/files#diff-5a17873aec4eae6b52b00959d8f9e17672912858f63181d39de8c3a713e90018R57-R71